pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -309,7 +317,14 @@ func (c *Reconciler) resolvePipelineState(
 		// We need the TaskRun name to ensure that we don't perform an additional remote resolution request for a PipelineTask
 		// in the TaskRun reconciler.
 		trName := resources.GetTaskRunName(pr.Status.TaskRuns, pr.Status.ChildReferences, task.Name, pr.Name)
-		fn := tresources.GetTaskFunc(ctx, c.KubeClientSet, c.PipelineClientSet, c.resolutionRequester, pr, task.TaskRef, trName, pr.Namespace, pr.Spec.ServiceAccountName)
+
+		// get verificationpolicies for trusted resources
```

Comment:
general comment about comments - only include a comment if it adds something you wouldn't know just from reading the code below it. in this case it does mention "trusted resources" which is new information but I'm not sure it makes the line much more clear 

the book "clean code" is a great resource on code clarity in general - https://gist.github.com/wojteklu/73c6914cc446146b8b533c0988cf8d29#comments-rules has a summary of some thoughts around comments